app/src/form/__test__/FormContainer.test.js

```diff
@@ -219,7 +219,53 @@ jest.mock('../../assets/administrations.db', () => {
   return 'data';
 });
 
-describe('FormContainer component', () => {
+describe('FormContainer component on save', () => {
+  test('should return values as null onSave callback if currentValues not defined', async () => {
+    const handleOnSave = jest.fn();
+
+    render(<FormContainer forms={exampleTestForm} onSave={handleOnSave} />);
+
+    await waitFor(() => {
+      expect(handleOnSave).toHaveBeenCalledTimes(1);
+      expect(handleOnSave).toHaveBeenCalledWith(null, expect.any(Function));
+    });
+  });
+
+  test('should handle onSave event and return refreshForm', async () => {
+    const handleOnSave = jest.fn();
+
+    const modifiedInitialValues = {
+      1: 'John',
```

Comment:
Add more tests to make this empty string

Reply:
Hi mas @dedenbangkit , I pushed an update for this